--- a.cmake	2022-08-30 16:38:01.663350469 +0200
+++ b.cmake	2022-08-30 16:41:23.411060498 +0200
@@ -122,7 +122,6 @@
 endif(VIENNALS_BUILD_SHARED_LIBS)
 
 # set the correct paths for installation
-set(CMAKE_INSTALL_PREFIX "${CMAKE_INSTALL_PREFIX}/${PROJECT_NAME}/")
-set(LOCAL_CONFIG_DIR "lib/cmake/${PROJECT_NAME}")
+set(LOCAL_CONFIG_DIR "lib/x86_64-linux-gnu/cmake/${PROJECT_NAME}")
 
 # Adding the install interface generator expression makes sure that the include
