--- a.cmake	2022-08-30 16:45:28.348278925 +0200
+++ b.cmake	2022-08-30 16:50:22.644935592 +0200
@@ -86,8 +86,7 @@
 add_library(${PROJECT_NAME} INTERFACE)
 
 # set the correct paths for installation
-set(CMAKE_INSTALL_PREFIX "${CMAKE_INSTALL_PREFIX}/${PROJECT_NAME}/")
-set(LOCAL_CONFIG_DIR "lib/cmake/${PROJECT_NAME}")
+set(LOCAL_CONFIG_DIR "lib/x86_64-linux-gnu/cmake/${PROJECT_NAME}")
 
 # Adding the install interface generator expression makes sure that the include
 # files are installed to the proper location (provided by GNUInstallDirs)
